DESCRIPTION

```diff
@@ -1,15 +1,44 @@
 Package: sageseqr
-Title: What the Package Does (One Line, Title Case)
+Title: Identify differentially expressed genes counts data.
 Version: 0.0.0.9000
 Authors@R: 
-    person(given = "First",
-           family = "Last",
+    person(given = "Kelsey",
+           family = "Montgomery",
            role = c("aut", "cre"),
-           email = "first.last@example.com",
-           comment = c(ORCID = "YOUR-ORCID-ID"))
-Description: What the package does (one paragraph).
-License: What license it uses
+           email = "kelsey.montgomery@sagebase.org",
+           comment = c(ORCID = "0000-0002-0126-8944"))
+Description: This workflow normalizes counts, performs model selection from known 
+    covariates using multivariate regression, automates outlier analysis using a 
+    customizable threshold and performs differential expression analysis for repeated
+    measures.
+License: MIT + file LICENSE
 Encoding: UTF-8
 LazyData: true
 Suggests: 
     testthat (>= 2.1.0)
+RoxygenNote: 7.1.0
+biocViews:
+    limma,
```

Comment:
The bioconductor packages don't go under the biocViews line, it's just there to tell R that it should look at both CRAN and bioconductor when installing the packages that are in Depends/Imports/etc.